TODO.txt

```diff
@@ -53,7 +53,7 @@ PG2:
   * add video shownotes to search
   * add stackoverflow to search
   * only search your locale plus english
-  * search internal pages, too (blog, etc)
+  * search internal pages, too
```

Comment:
this file could be converted into github issues, no?

Reply:
Yes, though I suspect most of it could just be deleted (I only dropped the "blog" mention here because I was grepping for "blog" to look for leftovers and it was polluting my results).

Reply:
I will create an issue to revisit this file and filter out what's no longer useful

Reply:
created as #1140